app/assets/scripts/components/ProjectForm.js

```diff
@@ -498,7 +472,129 @@ class ProjectForm extends React.Component {
         'monitoring-date': DateFieldFactory('Monitoring Date (Year) - تاريخ الرصد (عام)؛', 'Monitoring Date (Month) - تاريخ الرصد (شهر)؛'),
         'district': DistrictField,
         'marker': LocationField,
-        'currency': CurrencyField
+        'currency': CurrencyField,
+        'select-status': Dropdown(
+          'Project Status - وضع/ حالة المشروع',
+          'Select a status - يُرجى اختيار وضع محدد (حالة محددة)',
+          [
+            'Planned',
+            'Ongoing',
+            'Closed'
+          ],
+          [
+            ' مُخطط',
+            ' جاري/ مستمر',
+            ' مُغلق'
+          ]
+        ),
+        'select-ministry': Dropdown('Responsible Ministry - الوزارة المسؤولة', 'Select a Ministry',
+          ['Ministry of Agriculture and Land Reclamation', 'Ministry 2', 'Ministry 3'],
+          [' وزارة الزراعة واستصلاح الأراضي', '', '']
+        ),
+        'select-sds_indicator': Dropdown(
+          'SDS Goal - هدف استراتيجية التنمية المُستدامة',
+          'Select an SDS goal - يُرجى اختيار أحد أهداف استراتيجية التنمية المستدامة التى يتناولها المشروع',
+          [
+            'Pillar 1: Economic Development',
+            'Pillar 2: Energy',
+            'Pillar 3: Knowledge, Innovation and Scientific Research',
+            'Pillar 4: Transparency and Efficiency of Government Institutions',
+            'Pillar 5: Social Justice',
+            'Pillar 6: Health',
+            'Pillar 7: Education & Training',
+            'Pillar 8: Culture',
+            'Pillar 9: Environment',
+            'Pillar 10: Urban Development',
+            'Pillar 11: National Security and Foreign Policy',
+            'Pillar 12: Domestic Policy'
+          ],
+          [
+            ' المحور الأول: التنمية الاقتصادية',
+            ' المحور الثاني: الطاقة',
+            ' المحور الثالث: المعرفة والابتكار والبحث العلمي',
+            ' المحور الرابع: شفافية وكفاءة المؤسسات الحكومية',
+            ' المحور الخامس: العدالة الاجتماعية',
+            ' المحور السادس: الصحة',
+            ' المحور السابع: التعليم والتدريب',
+            ' المحور الثامن: الثقافة',
+            ' المحور التاسع: البيئة',
+            ' المحور العاشر: التنمية العمرانية',
+            '',
+            ''
```

Comment:
I removed the whitespace. The empty strings have to be there to match the number of items in each dropdown. They don't have equivalent translation strings.